src/client/java/teammates/client/scripts/helper/LoopHelper.java

```diff
@@ -0,0 +1,43 @@
+package teammates.client.scripts.helper;
+
+/**
+ * Utility class to be used with loops to print messages regularly.
```

Comment:
There is a `getCount()` method in this class that disagrees with this statement.